articles/model-anatomy.adoc

```diff
@@ -0,0 +1,678 @@
+include::../shared/issue-footnotes.adoc[]
+:serialization: ../serialization/serialization
+:m3: ../metametamodel/metametamodel
+:refarch: ../reference-architecture/reference-architecture
+:roadmap: ../roadmap/roadmap
+= Anatomy of a LionWeb Model
+:toc: preamble
+:toclevels: 3
+:sectnums:
+
+This article walks us through a LionWeb model:
+What it is, what's inside, how the inside looks, what the contents mean, and why they are that way.
+It is meant as an introduction to some of the details of LionWeb.
```

Comment:
Here maybe it helps to paint a picture in one or two words what the "some parts" mean. Is it technical or design level or user level. Maybe an "intended" audience. You do mention no knowledge is required but there are multipe perspectives of being involved in LionWeb.